### DISCUSSION TITLE
How to use pedantic mode With remark v13 (remark-parse v9.0.0)?

### DISCUSSION THREAD

**Discussion by Neos21** [remarkjs/remark#547] [Q&A] [answered]
remark v13.0.0 (remark-parse v9.0.0), `pedantic` options is no longer available.
But I want to use `pedantic` mode. Because I want to write emphasis with underscore `_` character.

(This is really just a personal preference. I'm Japanese, and I think underscore is more readable than asterisk in Japanese text...)

```
日本語_強調_日本語__強調__日本語
→ I want : 日本語<em>強調</em>日本語<strong>強調</strong>日本語
```

Please tell me how to use pedantic mode.
Or, please tell me a custom parser that can parse underscore without word-separation (like asterisk).

**Comment by wooorm** [comment, marked as answer]
This is no longer supported, because it doesn’t work in lots of places (e.g., here, on GitHub):

日本語_強調_日本語__強調__日本語

The removal is [documented in the changelog](https://github.com/remarkjs/remark/releases/tag/13.0.0). The suggestion is to change your markdown instead.

**Reply by Neos21** [reply]
I know why it changed.
But, still, I want to use underscore rather than asterisk.
Is there any other way? Can I create an original parser, etc.?

**Reply by wooorm** [reply]
Yes: https://github.com/remarkjs/remark/tree/main/packages/remark-parse#extending-the-parser. But I seriously strongly suggest not to for pedantic.

Have you seen discussions like this? https://talk.commonmark.org/t/bold-failure-in-chinese-sentence-because-of-the-usage-of-chinese-punctuation-is-different-from-english/3478/14
